### PR TITLE
[13.0][MIG] account_reconcile_restrict_partner_mismatch: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT20490

### CODE REVIEW

File: account_reconcile_restrict_partner_mismatch/tests/test_reconciliation.py [hunk 1]
```diff
@@ -2,44 +2,35 @@
...
 
 from odoo.exceptions import UserError
+from odoo.tests.common import SavepointCase
 
-from odoo.addons.account.tests.account_test_classes import AccountingTestCase
 
-
-class TestReconciliation(AccountingTestCase):
+class TestReconciliation(SavepointCase):
     def setUp(self):
```
Comment (yajo) [inline account_reconcile_restrict_partner_mismatch/tests/test_reconciliation.py]: You have to use `setUpClass` if you really want to benefit from `SavepointCase`

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by ernestotejeda** [comment]
Changes done

File: account_reconcile_restrict_partner_mismatch/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "name": "Reconcile restrict partner mismatch",
+    "summary": "Restrict reconciliation on receivable "
+    "and payable accounts to the same partner",
+    "version": "13.0.1.0.0",
+    "depends": ["account"],
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "http://www.github.com/OCA/account-reconcile",
+    "category": "Finance",
+    "license": "AGPL-3",
+    "data": ["report/account_move_lines_report.xml", "security/ir.model.access.csv"],
```
Comment (sergio-teruel) [inline account_reconcile_restrict_partner_mismatch/__manifest__.py:14]: For better display, put in separate lines

**Review by sergio-teruel — APPROVED** [review]

File: account_reconcile_restrict_partner_mismatch/__manifest__.py (hunk 2 above)
Comment (yajo) [reply account_reconcile_restrict_partner_mismatch/__manifest__.py:14]: Black did this, don't bother.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**